src/native/fs/fs_napi.cpp

```diff
@@ -284,63 +476,53 @@ struct FSWrapWorker : public FSWorker
 struct Stat : public FSWorker
```

Comment:
Might be worth commenting that now this op can actually do stat and lstat.
I know that having stat and lstat that are much similar is duplicating the code, but isn't it more straightforward? @jackyalbo @guymguym WDYT?

Reply:
Not sure what you mean by "more straighforward" here.
For example in open there is no lopen but instead there are flags to indicate how to treat symlinks.

Reply:
I have missed something in reading it, Looks good.